src/System.Diagnostics.Process/tests/ProcessThreadTests.cs

```diff
@@ -80,16 +79,21 @@ public void TestStartTimeProperty()
                     return;
                 }
 
-                if (ThreadState.Initialized != thread.ThreadState)
+                if (ThreadState.Initialized != thread.ThreadState && ThreadState.Ready != thread.ThreadState)
```

Comment:
Why exclude threads in a ready state?  Isn't that going to exclude many of them from being tested?